client/post-editor/editor-confirmation-sidebar/index.jsx

```diff
@@ -86,26 +89,25 @@ class EditorConfirmationSidebar extends React.Component {
 	getBusyButtonLabel( publishButtonStatus ) {
 		switch ( publishButtonStatus ) {
 			case 'update':
-				return this.props.translate( 'Updating...' );
+				return this.props.translate( 'Updating…' );
```

Comment:
:disappointed: 92 existing translations will be lost with this change.
If the string must change, the following option has already been translated **52** times:
`translate( 'Updating' )` *ES Score: 19*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).